aten/src/ATen/preprocess_declarations.py

```diff
@@ -31,46 +33,60 @@ def process_types_and_backends(option):
     # if specific pairs were not listed, then enumerate them
     # based on the backend and type attributes
     # if backend or type is not defined, it is assumed to be all of them
-    if 'backend_type_pairs' not in option:
+    if 'backend_types' not in option:
         backends = option.get('backends', default_backends)
         if isinstance(option.get('type_method_definition_dispatch'), dict):
             backends = option.get('type_method_definition_dispatch').keys()
         backends = set(backends)
 
-        types = option.get('types', all_types)
-
-        pairs = [[p, t] for p in backends for t in types]
+        backend_types = {}
+        for backend in backends:
+            if backend == 'QuantizedCPU':
+                backend_types[backend] = type_map['quantized']
+            else:
+                backend_types[backend] = option.get('types', all_types)
```

Comment:
wait, QInt8 is included in here?  Even though QInt8 is never included in this "backend"?  What's going on here?